docs/security/configuration.md

```diff
@@ -126,6 +126,26 @@ You need to add the certificate thumbprint setting on Windows so the server can
 | YAML                 | `CertificateThumbprint` |
 | Environment variable | `EVENTSTORE_CERTIFICATE_THUMBPRINT` |
 
+
+
+### Intermediate CA certificates
+
+Intermediate certificates are supported by loading them from a PEM or PKCS #12 bundle. The node's certificate should be first in the bundle, followed by the intermediates (intermediates can be in any order but it would be good to keep it from leaf to root as per the usual convention). The certificate chain is validated on startup with the node's own certificate.
```

Comment:
```suggestion
Intermediate certificates are supported by loading them from a [PEM](https://www.rfc-editor.org/rfc/rfc1422) or [PKCS #12](https://datatracker.ietf.org/doc/html/rfc7292) bundle. The node's certificate should be first in the bundle, followed by the intermediates (intermediates can be in any order but it would be good to keep it from leaf to root as per the usual convention). The certificate chain is validated on startup with the node's own certificate.
```

Reply:
**QUESTION:** Do we have some recommendations on when to use and when not to use intermediate CA certificates?

Could we also show the example bundle structure or add a link to how to configure such a bundle?

Reply:
It's always better to use one or more intermediate certificates as this is more secure, since the root CA is not directly used. People who deal with PKI should know this :)

We could show an example once the `es-gencert-cli` is updated, but not sure if those certs should be used in production and if it's wise to show it to people.

Reply:
I think that the audience trying to configure those certificates will not always be people dealing with PKI daily, so IMHO, it's always worth adding more context.

Regarding samples, I was more referring to somehow visualising or explaining the structure explained in the description.

Reply:
Think this info is more for people who are already using intermediate certificates and want to know if it's supported and how. For others, things regarding certificates will just stay as-is.  

Don't think we are in the business of telling how to do PKI, in the mongodb docs they are referring to [prior knowledge](https://docs.mongodb.com/manual/tutorial/configure-ssl/#prerequisites)

Not sure if I'm the right person to make something visual for the website/docs 😬